cmd/storage-rest-client.go

```diff
@@ -148,7 +125,7 @@ type storageRESTClient struct {
 // after verifying format.json
 func (client *storageRESTClient) call(method string, values url.Values, body io.Reader, length int64) (respBody io.ReadCloser, err error) {
 	if !client.connected {
-		return nil, errDiskNotFound
```

Comment:
Changing this might affect XL sets handling for network disconnected error - errDiskNotFound in unexpected ways

Reply:
reverted it to return errDiskNotFound in case of network error.

Also did this change:
```
 // Collection of basic errors.
 var baseErrs = []error{
-       errDiskNotFound,
        errFaultyDisk,
        errFaultyRemoteDisk,
 }
```

so that errDiskNotFound can be returned from reduceQuorumError() and it gets passed to the S3 layer so that `SlowDown` can be sent.

Reply:
some unit tests are failing

Reply:
I undid this change:
```
 // Collection of basic errors.
 var baseErrs = []error{
-       errDiskNotFound,
        errFaultyDisk,
        errFaultyRemoteDisk,
 }
```

as unit tests were failing (expecting errDiskNotFound)
Instead returning ErrSlowDown when the error is errDiskNotFound